src/strategies/goldfinch-voting-power/index.ts

```diff
@@ -0,0 +1,79 @@
+import { multicall } from '../../utils';
+import _strategies from '..';
+import { formatUnits } from '@ethersproject/units';
+
+export const author = 'blakewest';
+export const version = '0.1.0';
+
+const COMMUNITY_REWARDS = '0x0Cd73c18C085dEB287257ED2307eC713e9Af3460';
+const GFI = '0xdab396cCF3d84Cf2D07C4454e10C8A6F5b008D2b';
+
+const COMMUNITY_REWARDS_ABI = {
+  inputs: [
+    {
+      internalType: 'address',
+      name: 'owner',
+      type: 'address'
+    }
+  ],
+  name: 'totalUnclaimed',
+  outputs: [
+    {
+      internalType: 'uint256',
+      name: '',
+      type: 'uint256'
+    }
+  ],
+  stateMutability: 'view',
+  type: 'function'
+};
```

Comment:
```suggestion
const COMMUNITY_REWARDS_ABI = [ 'function totalUnclaimed(address owner) view returns (uint256)' ];
```


and same changes as in https://github.com/snapshot-labs/snapshot-strategies/pull/492 :)